src/index.js

```diff
@@ -236,6 +238,12 @@ export default class Carousel extends React.Component {
     this.getlockScrollEvents().unlockTouchScroll();
   }
 
+  loadPolyfills() {
+    if (typeof Element !== 'undefined') {
+      import('wicg-inert');
```

Comment:
lazy-loading isn’t supported in libraries we need to handle this on the app since we need a specific operation your bundler performs. In library code this won’t happen and will cause issues in some browsers. We need to fix this in `wicg-inert` itself

This will most likely work in some setups like next.js and modern browsers but not in all

Reply:
Ah very good point! Thanks! I've raised a PR in wicg-inert :)